curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/count-backwards-with-a-for-loop.md

```diff
@@ -10,7 +10,7 @@ forumTopicId: 16808
 
 A for loop can also count backwards, so long as we can define the right conditions.
 
-In order to count backwards by twos, we'll need to change our `initialization`, `condition`, and `final-expression`.
+In order to count backward by two, we'll need to change our `initialization`, `condition`, and `final-expression`.
```

Comment:
What about the following?
```suggestion
In order to count back by two each time, we'll need to change our `initialization`, `condition`, and `final-expression`.
```

Reply:
Would it be better to use more familiar jargon:
> In order to decrement by two each iteration...

Reply:
@matias-miller I have updated my suggestion to reflect @ShaunSHamilton's wording.  Please click the `Commit suggestion` button above and write an appropriate commit message for it.  Thanks.